Sources/OpenCombine/CurrentValueSubject.swift

```diff
@@ -22,16 +24,23 @@ public final class CurrentValueSubject<Output, Failure: Error>: Subject {
 
     /// The value wrapped by this subject, published as a new element whenever it changes.
     public var value: Output {
-        didSet {
-            send(value)
+        get {
+            defer {
+                _lock._unlock()
+            }
+            _lock._lock()
```

Comment:
I'm not sure Combine uses synchronized access to the `value` property though. Maybe that's why the test failed.

Reply:
I would guess that they at synchronise the access to value otherwise this would not be thread safe at all. The test failed because what tested is not something the the Combine/OpenCombine framework can synchronise but rather the user of the framework needs to synchronise.

Reply:
I've just checked the assembly, access to `value` is indeed not thread-safe.

Reply:
Hm, I tested the value in access in Combine and at least the thread sanitiser doesn't complain. What do you want me to do here?

Reply:
I removed the locks from within the get and set and the sanitiser didn't complain. So I removed it now.

Reply:
In this case, I think I'm ready to merge it, just waiting for you to push